src/main/java/com/nylas/Neural.java

```diff
@@ -0,0 +1,105 @@
+package com.nylas;
+
+import okhttp3.HttpUrl;
+
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Neural {
+	protected final String neuralPath = "neural";
+	protected final NylasAccount account;
+	protected final NylasClient client;
+	protected final String accessToken;
+	private static final Pattern imagePattern = Pattern.compile("[(']cid:(.)*[)']");
```

Comment:
Most of the time we have a convention of using UPPER_CASE for constant (static final) members, though I can see there are a couple of other exceptions.  I'm thinking it would be good to standardize.  What do you think?  Not a blocker.

Reply:
Totally agree! I like the upper snake casing for static final variables